### PR TITLE
Add filter and sort

### LINKED ISSUE #10: Add filter and sort
Added filter
Added Sort
Added delete feature on home page

### PR DESCRIPTION
**Pull request by Raghav888**
1. Added filter
2. Added Sort
3. Added delete feature on home page

### CODE REVIEW

File: src/components/Filterbox/Filterbox.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from "react";
+import { useNote } from "../../context/note-context";
 
 import "./filterbox.css";
 export const Filterbox = (props) => {
+  const { noteListDisptach, noteListState } = useNote();
   return (
```
Comment (sushil-mod) [inline src/components/Filterbox/Filterbox.js:7]: your code is also correct just a suggestion :-  you can desturcture noteListState here like const { sortbydate , sortbypriority , tags}= noteListState

**Review by sushil-mod — APPROVED** [review]
just a suggestion ,rest code is good

File: src/utils/noteListReducer.js [hunk 2]
```diff
@@ -16,10 +16,44 @@ export const noteListReducer = (state, action) => {
...
+          (item) => item !== action.payload.value
+        );
+        return { ...state, tags: newTag };
+      }
+    case "DELETE_NOTE":
+      (async () => {
+        await axios.delete(
+          `https://my-json-server.typicode.com/Raghav888/mynoteappAPI/notes/${action.payload.value}`
+        );
+      })();
```
Comment (shaksham08) [inline src/utils/noteListReducer.js:50]: Nice use of IIFT function

**Review by shaksham08 — APPROVED** [review]
Overall code looks well structured.

**Pull request merged**